java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/utils/PhysicalNodeUtil.scala

```diff
@@ -93,15 +93,11 @@ object PhysicalNodeUtil {
       val orderExpr = orders.GetOrderExpression(i)
       val colIdx = SparkColumnUtil.resolveOrderColumnIndex(orderExpr, windowAggNode.GetProducer(0))
       val column = SparkColumnUtil.getColumnFromIndex(inputDf, colIdx)
-      // TODO: Support reading is_asc from physical node, currently use asc by default
-      orderByCols += column.asc
-      /*
-      if (orders.is_asc()) {
+      if (orderExpr.is_asc()) {
```

Comment:
Add the unit tests to verify the config works.

Reply:
Add the unit tests in next PR

Reply:
Have solve in https://github.com/4paradigm/OpenMLDB/pull/296